File: lab-2/src/moves/CrossChop.java
package moves;
import ru.ifmo.se.pokemon.*;
public class CrossChop extends PhysicalMove{
    public CrossChop(){
        super(Type.FIGHTING, 100, 80);
    }


    protected void  applySelfEffects(Pokemon p){
        p.setMod(Stat.SPEED, 4);
    }

    public String describe(){
        return "is using Cross Chop";
    }


}

File: lab-2-kotlin/src/moves/CrossChop.kt
package moves

import ru.ifmo.se.pokemon.*

class CrossChop : PhysicalMove(Type.FIGHTING, 100.0, 80.0) {
    override fun applySelfEffects(p: Pokemon) {
        p.setMod(Stat.SPEED, 4)
    }

    public override fun describe(): String {
        return "is using Cross Chop"
    }
}